tests/scalers/etcd/etcd_cluster_test/etcd_cluster_test.go

```diff
@@ -0,0 +1,182 @@
+	EtcdName         string
+}
+
+const (
+	deploymentTemplate = `apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{.DeploymentName}}
+  namespace: {{.TestNamespace}}
+spec:
+  selector:
+    matchLabels:
+      app: {{.DeploymentName}}
+  replicas: 0
+  template:
+    metadata:
+      labels:
+        app: {{.DeploymentName}}
+    spec:
+      containers:
+      - name: my-app
+        image: nginx
+        imagePullPolicy: IfNotPresent
+        ports:
+        - containerPort: 80
+`
+	scaledObjectTemplate = `apiVersion: keda.sh/v1alpha1
+kind: ScaledObject
+metadata:
+  name: {{.ScaledObjectName}}
+  namespace: {{.TestNamespace}}
+spec:
+  scaleTargetRef:
+    name: {{.DeploymentName}}
+  pollingInterval: 30
+  cooldownPeriod: 5
+  advanced:
+    horizontalPodAutoscalerConfig:
+      name: keda-hpa-etcd-scaledobject
+      behavior:
+        scaleDown:
+          stabilizationWindowSeconds: 5
+  triggers:
+    - type: etcd
+      metadata:
+        endpoints: {{.EtcdName}}-0.etcd-headless.{{.TestNamespace}}:2379,{{.EtcdName}}-1.etcd-headless.{{.TestNamespace}}:2379,{{.EtcdName}}-2.etcd-headless.{{.TestNamespace}}:2379
+        key: var
+        value: '1.5'
+        watchProgressNotifyInterval: '10'
+`
+	insertJobTemplate = `apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{.JobName}}
+  namespace: {{.TestNamespace}}
+spec:
+  ttlSecondsAfterFinished: 0
+  template:
+    spec:
+      containers:
+      - name: etcd
+        image: gcr.io/etcd-development/etcd:v3.4.20
+        imagePullPolicy: IfNotPresent
+        command:
+        - sh
+        - -c
+        - "/usr/local/bin/etcdctl put var 9 --endpoints=http://{{.EtcdName}}-0.etcd-headless.{{.TestNamespace}}:2380,http://{{.EtcdName}}-1.etcd-headless.{{.TestNamespace}}:2380,http://{{.EtcdName}}-2.etcd-headless.{{.TestNamespace}}:2380"
+      restartPolicy: Never
+  backoffLimit: 4
+`
+	deleteJobTemplate = `apiVersion: batch/v1
+kind: Job
+metadata:
+  name: {{.JobName}}
+  namespace: {{.TestNamespace}}
+spec:
+  ttlSecondsAfterFinished: 0
+  template:
+    spec:
+      containers:
+      - name: etcd
+        image: gcr.io/etcd-development/etcd:v3.4.20
+        imagePullPolicy: IfNotPresent
+        command:
+        - sh
+        - -c
+        - "/usr/local/bin/etcdctl put var 0 --endpoints=http://{{.EtcdName}}-0.etcd-headless.{{.TestNamespace}}:2380,http://{{.EtcdName}}-1.etcd-headless.{{.TestNamespace}}:2380,http://{{.EtcdName}}-2.etcd-headless.{{.TestNamespace}}:2380"
+      restartPolicy: Never
+  backoffLimit: 4
+`
+)
+
+func TestScaler(t *testing.T) {
+	// setup
+	t.Log("--- setting up ---")
+	// Create kubernetes resources
+	kc := GetKubernetesClient(t)
+
+	// Create kubernetes resources for testing
+	data, templates := getTemplateData()
+	CreateKubernetesResources(t, kc, testNamespace, data, templates)
+
+	// Create Etcd Cluster
+	etcd.InstallCluster(t, kc, testName, testNamespace)
+
+	testActivation(t, kc, data)
+	testScaleUp(t, kc, data)
+	testScaleDown(t, kc)
```

Comment:
Use Scale in/out instead of up/down please, we are horizontal autoscaling (I know other test were wrong too but they are already fixed)

Reply:
done